src/pages/tasks/common/hooks/useStop.ts

```diff
@@ -24,6 +27,18 @@ export function useStop() {
   return (task: Task) => {
     toast.processing();
 
+    const logs = parseTimeLog(task.time_log);
+
+    if (logs[logs.length - 1][0] && logs[logs.length - 1][0] > dayjs().unix()) {
+      logs[logs.length - 1][1] = logs[logs.length - 1][0] + 1;
+
+      task.time_log = JSON.stringify(logs);
```

Comment:
This really looks like 🍝. Is it possible to simplify & perhaps update naming?

Reply:
It's actually pretty simple, but I'm sorry I missed naming. It is now aptly named and I think there is no more confusion.